tests/pw/README.MD

```
@@ -1,389 +1,574 @@
+npm run docker:setup
+```
 
-<br/>
+---
 
-:radio_button: Run tests in UI Mode
+## Access Points
 
-    npx playwright test --ui
+| Service | URL | Credentials |
+|---------|-----|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)